docs/getting-started/upgrade/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Upgrade Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/upgrade/"
+---
+
+You can manage upgrades to Shopify App CLI with the package manager for your platform.
+
+### Brew (MacOS)
+
+```console
+$ brew update && brew upgrade shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+```console
+$ sudo apt update
+$ sudo apt upgrade shopify-cli
+```
+
+### RPM (CentOS, Fedora, Red Hat, SUSE)
+
+```console
+$ sudo yum upgrade shopify-cli
```

Comment:
As per the comment above, the `shopify-cli` RPM package won't be in a repo for a while, so the upgrade will be the same as install, i.e., download a newer file and do `sudo yum install shopify-cli-x.y.z.rpm`.